core/src/main/java/org/elasticsearch/action/percolate/PercolateRequest.java

```diff
@@ -244,13 +234,9 @@ public PercolateRequest onlyCount(boolean onlyCount) {
         return this;
     }
 
-    BytesReference docSource() {
-        return docSource;
-    }
-
     @Override
     public ActionRequestValidationException validate() {
-        ActionRequestValidationException validationException = super.validate();
+        ActionRequestValidationException validationException = null;
```

Comment:
These are some pre checks that always were there. This method was only changed because the base class used to be different and now I can't invoke `super.validate()`